modules/integration/tests-common/web-ui-pages/src/main/java/org/wso2/iot/integration/ui/pages/devices/EnrollDevicePage.java

```diff
@@ -0,0 +1,89 @@
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.wso2.iot.integration.ui.pages.UIUtils;
+import org.wso2.iot.integration.ui.pages.UIElementMapper;
+import org.wso2.iot.integration.ui.pages.samples.ConnectedCupDeviceTypeViewPage;
+
+import java.io.IOException;
+
+/**
+ * Device Enrollment page.
+ * This page lists the all device types which are currently installed in the IOT server.
+ */
+public class EnrollDevicePage {
+
+    private WebDriver driver;
+    private UIElementMapper uiElementMapper;
+    private Log log = LogFactory.getLog(EnrollDevicePage.class);
+
+    public EnrollDevicePage(WebDriver driver) throws IOException {
+        this.driver = driver;
+        this.uiElementMapper = UIElementMapper.getInstance();
+
+        WebDriverWait webDriverWait = new WebDriverWait(driver, UIUtils.webDriverTimeOut);
+        if (!webDriverWait.until(ExpectedConditions.titleContains("Device Types | IoT Server"))) {
+            throw new IllegalStateException("This is not the Device Enrollment page");
+        }
+    }
+
+    /**
+     * This method checks whether the Connected cup device is installed. If the device is installed correctly, it is
+     * displayed in this page.
+     * @return : True if the device is visible in the page. False otherwise.
+     */
+    public boolean isInstalled() {
+        boolean check = UIUtils.isElementPresent(log, driver, By.xpath(
+                uiElementMapper.getElement("iot.sample.connectedcup.xpath")));
+        if (check) {
+            WebElement deviceDiv = driver.findElement(By.xpath(
+                    uiElementMapper.getElement("iot.sample.connectedcup.xpath")));
+            WebElement tryBtn = deviceDiv.findElement(By.tagName("button"));
+            return tryBtn.isDisplayed();
+        }
+        return false;
+    }
+
+    /**
+     * Method to perform the navigation to Device type view page of the Connected cup.
+     * @return : The corresponding Device type view page. Null, if the element is not visible.
+     */
+    public ConnectedCupDeviceTypeViewPage gotoConnectedCupDeviceTypeViewPage() throws IOException {
+        boolean check = UIUtils.isElementPresent(log, driver, By.xpath(
+                uiElementMapper.getElement("iot.sample.connectedcup.xpath")));
+        if (check){
+            WebElement deviceDiv = driver.findElement(By.xpath(
+                    uiElementMapper.getElement("iot.sample.connectedcup.xpath")));
+            WebElement tryBtn = deviceDiv.findElement(By.tagName("button"));
+            tryBtn.click();
+            return new ConnectedCupDeviceTypeViewPage(driver);
+        } else {
+            log.error("Element not found...........................");
```

Comment:
Can we make the error log standerd

Reply:
This is fixed in the commit https://github.com/wso2/product-iots/pull/1454/commits/399136daae1b5183134e5b3da6c5fef9fa633f47